client/src/components/v3/status/WMStatus.tsx

```diff
@@ -74,7 +74,7 @@ export default function WMStatus(props: WMStatusProps) {
       cadence: 0,
       heartRate: 0,
       reedVelocity: 1,
```

Comment:
Reed velocity shows 1 dp here but 2 dp on the main page. Would be good to be consistent, make them both 2 dp. Same applies to GPS speed.